test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -17,6 +18,25 @@ Describe "Measure-Object" -Tags "CI" {
         $($testObject | Measure-Object).Count | Should Be $testObject.Length
     }
 
+    It "Should calculate Standard Deviation" {
+        $actual = ($testObject | Measure-Object -Average -StdDeviation)
+        # We check this way since .StdDeviation returns a double value
+        [Math]::abs($actual.StdDeviation - 1.52752523165195) | Should -BeLessThan .00000000000001
```

Comment:
We need to trust this. I think we need to add a comment from where we took  `1.52752523165195`. Below too.

Reply:
Should we just add a comment that says we calculated it?